lib/constants/music_list.dart

```diff
@@ -0,0 +1,29 @@
+List musicList = [
```

Comment:
can you comment what this does explicitly so that Owen and I can learn from this technique of display?